integrations/cdi/hibernate-cdi/src/main/java/io/helidon/integrations/cdi/hibernate/HibernateJtaPlatform.java

```diff
@@ -29,13 +29,13 @@
  * An {@link AbstractJtaPlatform} that is an {@link ApplicationScoped}
  * CDI managed bean that supplies {@link TransactionManager} and
  * {@link UserTransaction} instances that are supplied to it at
- * {@linkplain #CDISEJtaPlatform(TransactionManager, UserTransaction)
+ * {@linkplain #HibernateJtaPlatform(TransactionManager, UserTransaction)
  * construction time}.
  *
  * @see AbstractJtaPlatform
  */
 @ApplicationScoped
-public class CDISEJtaPlatform extends AbstractJtaPlatform {
+public class HibernateJtaPlatform extends AbstractJtaPlatform {
```

Comment:
`JtaPlatform` is inherently a Hibernate construct, so renaming this to `HibernateJtaPlatform` is redundant.

Reply:
I will change it back and refresh the PR

Reply:
Reverted